sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -282,316 +458,356 @@ q.parallel_for(nd_range<2>(G, L), [=](nd_item<2> item)
-|`msize`, `nsize`, `ksize`|  validation, default values, general query| if the TPU implementation supports a discrete number of element sizes, each of these members is non-zero, and the value tells one of the supported element sizes. By contrast, if the TPU supports a continuous number of element sizes, each of these members has the value zero
-|`atype`, `btype`, `accumulatortype`| validation, default values, general query| indicates the types supported in the combination
-|`combinations`    | validation, default values, general query| tells the set of supported matrix sizes and types according to the template parameters that are provided. In the "general query" form, the user provides only the TPU type, so the combinations array contains all supported tile sizes and element types for that TPU. In the "default values" form, the user provides the TPU type and element types, so the combinations array contains only those supported matrix sizes and element types that match those element types on that TPU. In the "validation" form, the user provides the TPU type, element types, and element sizes so only this specific combination is returned in the combinations array. 
-|`num_combinations`|  validation, default values, general query|indicates number of combinations supported by the TPU implementation which corresponds to the size of the `combinations` array
+| Member/type alias in `matrix_params` | Description
+|`M`|when no sizes are provided by the user, indicates the suggested
+default size for M; usually this corresponds to the maximum size the
+implementation supports. In validation mode, where the user does
+provide sizes, this is the same value M that the user provides if M is
+supported by the implementation
+|`N`|when no sizes are provided by the user, indicates the suggested
+default size for N; usually this corresponds to the maximum size the
+implementation supports. In validation mode, where the user does
+provide sizes, this is the same value N that the user provides if N is
+supported by the implementation
+|`K`| when no sizes are provided by the user, indicates the suggested
+default size for K; usually this corresponds to the maximum size the
+implementation supports. In validation mode, where the user does
+provide sizes, this is the same value K that the user provides if K is
+supported by the implementation
+|`template <typename Group, layout Layout> using joint_matrix_a;`| type
+alias for `joint_matrix` for matrix A
+|`template <typename Group, layout Layout> using joint_matrix_b;`| type
+alias for `joint_matrix` for matrix B
+|`template <typename Group> using joint_matrix_accumulator;`| type
+alias for `joint_matrix` for matrix accumulator
```

Comment:
Please adjust the column width in this table to avoid line wrap.

Reply:
The column widths were not changed in your recent commit bf8e00c.  The column widths are defined by the "col" option in the table definition.  For example, change the line:

> [frame="none",options="header"]

to something like:

> [frame="none",options="header",cols="30%,70%"]

The percentages indicate the fraction of each column's width in the table (the numbers must add to 100).  You'll have to play around with the numbers to avoid the line wrap for the `template<typename Group>` or `template<typename Group, layout Layout>`.  To check your results, look at the HTML render in GitHub.

Reply:
Everything looks good, but this one comment has not been addressed yet.

Reply:
Fixed in ee282500c36070319828ff5d62e585f5fa5e887d